src/etc/inc/plugins.inc.d/ntpd.inc

```diff
@@ -330,13 +330,15 @@ function ntpd_configure_do($verbose = false)
     $noselect = isset($config['ntpd']['noselect']) ? explode(' ', $config['ntpd']['noselect']) : [];
     $prefer = isset($config['ntpd']['prefer']) ? explode(' ', $config['ntpd']['prefer']) : [];
     $iburst = isset($config['ntpd']['iburst']) ? explode(' ', $config['ntpd']['iburst']) : [];
+    $have_pools = false;
```

Comment:
Needs to go to the `foreach()`, originally contemplated using it for something more global for https://github.com/opnsense/core/pull/7760  but my brain got hurt once again, notably for the reasons mentioned in https://github.com/opnsense/core/pull/7760#discussion_r1738053007

That PR will need to wait for later.